test/itemlist_formaction.cpp

```diff
@@ -460,8 +460,80 @@ TEST_CASE("OP_BOOKMARK works properly", "[itemlist_formaction]") {
 
 }
 
-TEST_CASE("process_op(OP_EDITFLAGS)", "[itemlist_formaction]") {
-	//NOTIMPL
+TEST_CASE("OP_EDITFLAGS arguments are added to an item's flags", "[itemlist_formaction]") {
```

Comment:
The flags aren't *added*—they're *set*, i.e. the new set of flags completely replaces the old one.

Come to think of it, there probably should be a test case demonstrating that flags are overwritten, not just appended or something. Can you please add one?